internals/pixels-mocks/src/lib.rs

```diff
@@ -1,57 +1,19 @@
 #![deny(clippy::all)]
 
-pub struct Rwh;
+pub struct Window;
 
-unsafe impl raw_window_handle::HasRawWindowHandle for Rwh {
-    fn raw_window_handle(&self) -> raw_window_handle::RawWindowHandle {
-        #[cfg(target_os = "macos")]
-        return raw_window_handle::RawWindowHandle::AppKit(
-            raw_window_handle::AppKitWindowHandle::empty(),
-        );
-        #[cfg(any(
-            target_os = "linux",
-            target_os = "dragonfly",
-            target_os = "freebsd",
-            target_os = "netbsd",
-            target_os = "openbsd",
-        ))]
-        return raw_window_handle::RawWindowHandle::Wayland(
-            raw_window_handle::WaylandWindowHandle::empty(),
-        );
-        #[cfg(target_os = "windows")]
-        return raw_window_handle::RawWindowHandle::Win32(
-            raw_window_handle::Win32WindowHandle::empty(),
-        );
-        #[cfg(target_os = "ios")]
-        return raw_window_handle::RawWindowHandle::UiKit(
-            raw_window_handle::UiKitWindowHandle::empty(),
-        );
+impl raw_window_handle::HasWindowHandle for Window {
+    fn window_handle(
+        &self,
+    ) -> Result<raw_window_handle::WindowHandle<'_>, raw_window_handle::HandleError> {
+        unimplemented!()
```

Comment:
Interesting! This should be ok because we don't want the tests to run against `pixels-mocks`, we just need to ensure they build. N.b. #80

Reply:
Part of the reason I did this is because all of the `::empty` methods on the various raw handle types got removed, so now you need to provide atleast a non-null pointer when constructing them. We could certainly try to make them runnable, though that might get involved.